pkgs/applications/networking/mailreaders/evolution/evolution-ews/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchurl, gnome3, cmake, gettext, intltool, pkg-config, evolution-data-server
+, sqlite, gtk3, webkitgtk, libgdata }:
+
+stdenv.mkDerivation rec {
+  pname = "evolution-ews";
+  version = "3.36.4";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/${pname}/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "0zfq02h3r1qbxak04i49564q4s2ykvkgcyc3krjgndan9lq3kvvn";
+  };
+
+  nativeBuildInputs = [ cmake gettext intltool pkg-config ];
+
+  buildInputs = [
+    evolution-data-server gnome3.evolution
+    sqlite libgdata
+    gtk3 webkitgtk
+  ];
+
+  # Building with libmspack as reccommended: https://wiki.gnome.org/Apps/Evolution/Building#Build_evolution-ews
+  cmakeFlags = [
+    "-DWITH_MSPACK=ON"
```

Comment:
This should be on by default IIUC: https://gitlab.gnome.org/GNOME/evolution-ews/-/blob/ad2d6373b8b6c3b83790ceb454256b6547256455/CMakeLists.txt#L149

But I wonder how it can build without `libmspack` being added to deps.

Reply:
Right, it does not build. Could you add the dependency?

Reply:
Done, dependency added